VMEncryption/main/Common.py

```diff
@@ -113,6 +113,12 @@ class CommonVariables:
     MigrateKey = 'MigrateFlag'
     MigrateValue = 'Migrate'
 
+    """
+    value for KeyStoreTypeKey could be MHSM
```

Comment:
value for KeyStoreTypeKey could be `ManagedHSM`  (instead of `MHSM)` just to convey the right string value in the comment.

Reply:
Probably you can also add an comment saying that , the parameter would accept `Keyvault` as the value in future. But this is optional.

Reply:
As per discussion with Jack, I will add a comment that this flag is for testing purposes only and will be removed before public MHSM release